kitsune/kpi/static/kpi/js/components/Chart.es6.js

```diff
@@ -0,0 +1,182 @@
+/* globals d3:false, $:false, _:false */
+/* jshint esnext: true */
+
+export default class Chart {
+  constructor($container, options) {
+    let defaults = {
+      chartColors: ['#EE2820', '#F25743', '#F58667', '#F9B58B', '#FDE4AF', '#E3F1B6', '#AADA9F', '#72C489', '#39AD72', '#00975C'],
+      axes: {
+        xAxis: {
+          labels: [],
+          labelOffsets: { x: 15, y: 23 }
+        },
+        yAxis: {
+          labels: [],
+          labelOffsets: { x: 5, y: 23 }
+        },
+        getPosition: (position, axis, index) => {
```

Comment:
So this one I wrote as an arrow function because otherwise the `this` context gets messed up. The way it's written now, `this` represents the Chart (so I have access to `this.gridSize`). If I rewrite it the other way, the `this` context becomes the `axes` object rather than the Chart (which doesn't have a reference to the gridSize).